app/views/vatDetails/details.scala.html

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  *@
 
-@(user: models.User, details: models.VatDetailsModel)(implicit request: Request[_], messages: Messages, appConfig: config.AppConfig)
+@(user: models.User, details: models.VatDetailsModel, serviceInfo: Html = Html(""))(implicit request: Request[_], messages: Messages, appConfig: config.AppConfig)
 
-  @views.html.main_template(title = messages("vatDetails.title"), bodyClasses = None, appConfig = appConfig) {
+  @views.html.main_template(title = messages("vatDetails.title"), bodyClasses = None, appConfig = appConfig, serviceInfo = serviceInfo) {
```

Comment:
Can you add a unit test for this in the spec please? i.e. if some html is passed in that it is rendered, don't need to check all of the content necessarily